roles/bootstrap-os/tasks/main.yml

```diff
@@ -40,6 +42,9 @@
 - include_tasks: bootstrap-opensuse.yml
   when: os_family == "opensuse"
 
+- include_tasks: bootstrap-clear.yml
```

Comment:
Must be `bootstrap-clearlinux.yml` here too

Reply:
*facepalm* Fixed it.